docs/sdk/serverless-signing/broadcast-and-collect.md

```diff
@@ -3,19 +3,24 @@ import FeedbackComponent from "@site/src/pages/feedback.md";
 # Broadcast and Collect Within an Action
 
 :::info
-Only available on the `test` network
+    Only available on the Lit networks:
+        - `datil`
+        - `datil-test`
+        - `datil-dev` 
 :::
```

Comment:
I think we can remove this. The original callout was because this was a brand new feature that hadn't been deployed to habanero/manzano yet, but we're past that now